### PR TITLE
bump expensify-common to 2.0.108

### PR DESCRIPTION
**Pull request by s77rt**
cc @dangrous @ntdiary 

### Details
bump expensify-common to 2.0.108 to include a more performant regex https://github.com/Expensify/expensify-common/pull/822

### Related Issues

https://github.com/Expensify/App/issues/52475

### Manual Tests
1. Type `test_test`
3. Verify the app doesn't crash

### CODE REVIEW

**Comment by s77rt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by tomekzaw — DISMISSED** [review]
LGTM, let's wait for CIs.

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "typescript": "~5.3.3"
   },
   "peerDependencies": {
-    "expensify-common": ">=2.0.106",
+    "expensify-common": ">=2.0.108",
```
Comment (tomekzaw) [inline package.json:103]: No need to bump this in my opinion, but if you do bump this, please also update README.md accordingly.

**Review by tomekzaw — APPROVED** [review]
Thanks for the PR!

**Pull request merged**